### PR TITLE
Changes to vestri-data to include the payment changes.

### PR DESCRIPTION
**Pull request by aChanEP**
Description:

Updated vestri data to include the latest payment changes and deleted old payments entities. 

1.) Deleted `payment_gatways.xml` and all its associations.
2.) Add `payment_providers.xml`.
3.) Linked all OOTB demo payment provider configurations to vestri stores in different regions.
3.) Changed pathing in `importconfiguration.xml` and `liquibase.changelog.xml` to match that of the OOTB mobee dataset.

Linting:

- [x] No linting errors

Component Updates:

- [ ] Component package requires update on npm [@elasticpath/store-components](https://www.npmjs.com/package/@elasticpath/store-components)

Tests:

- [ ] E2E tests (npm test run with `e2e`)
- [x] Manual tests

-- imported the dataset locally with the new `payment_providers.xml` and was able to view and go through checkout flow.

Documentation:

- [ ] Requires documentation updates

### CODE REVIEW

File: data/ep-blueprint-data/importexport/configuration/importconfiguration.xml [hunk 1]
```diff
@@ -143,7 +118,7 @@
   <packager type="NONE"/>
   <retrieval>
     <method>FILE</method>
-    <source>${data.population.data.directory}/ep-blueprint-data/importexport</source>
+    <source>${data.population.data.directory}/ep-blueprint-data/ep-blueprint-data/importexport/</source>
```
Comment (shaunmaharaj) [inline data/ep-blueprint-data/importexport/configuration/importconfiguration.xml]: is the nested directory with the same name intentional?

File: data/ep-blueprint-data/importexport/stores.xml [hunk 2]
```diff
@@ -143,5 +166,21 @@
...
+        <payment_provider_configuration_guid>credit-unsupported-config</payment_provider_configuration_guid>
+        <payment_provider_configuration_guid>reverse-unsupported-config</payment_provider_configuration_guid>
+        <payment_provider_configuration_guid>modify-unsupported-config</payment_provider_configuration_guid>
+        <payment_provider_configuration_guid>modify-and-cancel-unsupported-config</payment_provider_configuration_guid>
+        <payment_provider_configuration_guid>reserve-unsupported-config</payment_provider_configuration_guid>
+        <payment_provider_configuration_guid>cancel-reserve-fail-config</payment_provider_configuration_guid>
+        <payment_provider_configuration_guid>email-required-payment-provider-config</payment_provider_configuration_guid>
+    </payment_provider_configurations>
 </store>
 </stores>
```
Comment (shaunmaharaj) [inline data/ep-blueprint-data/importexport/stores.xml]: please add new line

File: data/ep-blueprint-data/liquibase-changelog.xml [hunk 3]
```diff
@@ -8,7 +8,7 @@
 		<comment>Initial import of ep vestri blueprint data.</comment>
 
 		<customChange class="com.elasticpath.datapopulation.importexport.ImportExportChange">
-			<param name="dataSet" value="ep-blueprint-data" />
+			<param name="dataSet" value="ep-blueprint-data/ep-blueprint-data" />
```
Comment (shaunmaharaj) [inline data/ep-blueprint-data/liquibase-changelog.xml]: nested directory?

**Comment by shaunmaharaj** [comment]
@aChanEP Does this load successfully on a 7.6 environment as well? Just curious as the education team and demo team pull the data zip for some demos. If it doesn't load, then we might need to zip the updated data files into a new zip (ex. **data/ep-blueprint-data_8.0.0.zip**) or rename the old one (ex. **data/ep-blueprint-data_7.6.zip**) and push so to keep the existing one available for older users.

**Comment by aChanEP** [comment]
Rebased this branch on top of mono repo changes.  Have also updated the data file to contain two versions of the vestri data -- A zip containing 7.6.0 data and renamed updated data with 8.0.0 in the name

**Pull request merged**